dbms/src/TiDB/Schema/SchemaBuilder.cpp

```diff
@@ -398,32 +398,42 @@ void SchemaBuilder<Getter, NameMapper>::applySetTiFlashReplica(DatabaseID databa
             return;
         }
 
-        applyDropTable(database_id, table_id, "SetTiFlashReplica-0");
+        updateTiFlashReplicaNumOnStorage(database_id, table_id, storage, table_info);
+        // FIXME: Do not drop the table under release-7.5 branch, need more testing
+        // applyDropTable(database_id, table_id, "SetTiFlashReplica-0");
```

Comment:
When the tiflash replica is set to 0, do not drop the IStorage instance under release-7.5 branch